docs/modules/ROOT/pages/operations/browser-sso.adoc

```diff
@@ -1,21 +1,21 @@
 :description: Single Sign-On (SSO) for Neo4j Browser.
 
-
+[role=enterprise-edition]
 [[browser-sso]]
 = Browser Single Sign-On
 
-Neo4j Browser provides basic support for some Single Sign-On (SSO) providers under specific Enterprise environment configurations.
-Contact Neo4j Professional Services for additional assistance (see link:https://neo4j.com/professional-services/[_https://neo4j.com/professional-services/_^] for more information about Professional Services).
-Your organization needs an existing compatible SSO provider (local or external) already configured.
+Neo4j Browser provides support for Single Sign-On (SSO) providers under specific Enterprise environment configurations when used with (self-hosted) Neo4j Server v4.4 and later.
+Additionally, you need a compatible SSO provider (local or external) already configured.
+For more information on the configuration, see link:https://neo4j.com/docs/operations-manual/current/authentication-authorization/sso-integration/#auth-sso-parameters[Operations Manual -> OIDC configuration settings^].
 
 [NOTE]
 ====
-Single Sign-On is _only_ available with the Enterprise Edition of Neo4j and requires engagement from Neo4j Professional Services to configure.
-General availability of SSO for Enterprise customers that does not require Professional Services' involvement is planned in a future release.
+For deployments other than with self-hosted Neo4j Server v4.4+, SSO can be configured with a separate OAuth plugin which requires engagement from Professional Services.
```

Comment:
Don't have anything to add to this myself other than to triple check with PS they are happy how this is presented. Mainly to ensure we don't drive too many people to enquire about the pre 4.4 SSO solution.